resources/build/build-utils.sh

```diff
@@ -41,12 +41,24 @@ function findRepositoryRoot() {
     # None of the answers are 100% correct for cross-platform
     # On macOS, requires coreutils (`brew install coreutils`)
 
+    # Return if readonly vars already set
+    if [ -v KEYMAN_ROOT ]; then
+        return 0;
+    fi
```

Comment:
When `android/build.sh` and `android/KMAPro/build.sh` source `build-utils.sh`, this was causing errors trying to assign a readyonly var a second time.

Reply:
argh, and the test fails on the mac BA. Back to the drawing board :/

Reply:
I don't know the bash-fu to properly do this.
Is it feasible to update the mac BA Bash to a modern version (4.0+)?

Reply:
I reverted this test, and ended up adding an assumption [comment](https://github.com/keymanapp/keyman/blob/4b153468f9ecefa863e98957c66b7dde76d64f79/resources/build/build-download-resources.sh#L12)

> `# Assumption: parent script that sources this already has START STANDARD BUILD SCRIPT INCLUDE`